README.md

```diff
@@ -1,60 +1,81 @@
 
-## Hardware Requirements
-* You need a 3D graphics card. (NVIDIA GeForce 8xxx and higher, ATI Radeon HD 4xxx and higher or Intel HD 3000 and higher.)
-* You should have a CPU that's running at 1GHz or better.
-* You'll need at least 512 MB of free VRAM (video memory).
-* Disk space: 400MB
-* Ideally, you want a joystick with at least 6 buttons.
+## System recommendations for running SuperTuxKart
+
+To run SuperTuxKart, make sure that your computer's specifications are equal or higher than the following recommended specifications:
+
+### Desktop
+
+To run SuperTuxKart on your desktop PC, check that your system specifications are equal to, or newer than the following system specifications:
+
+* CPU: Intel Pentium G2020 or AMD A4-4000 (and newer)
+* Graphics Card: NVIDIA GeForce GT 520 or AMD Radeon HD 6450, or Intel HD Graphics (and newer) - OpenGL >=3.1
+* System memory: 4 GB
+* Storage: 800 MB
+* Monitor resolution: 1024*768 (and higher)
+* Mouse and keyboard (a gamepad is recommended)
+
+### Laptop
+
+To run SuperTuxKart with the Antarctica engine on your laptop, check that your laptop's specifications are equal to, or higher than these recommendations:
+
+* Manufacture date: Newer than September 2013
+* OpenGL >= 3.1
+* System memory: 2 GB
+* Storage: 800 MB
+* Mouse and keyboard (a gamepad is recommended)
 
 ## License
-This software is released under the GNU General Public License (GPL) which can be found in the file [`COPYING`](/COPYING) in the same directory as this file. Information about the licenses for artwork are contained in `data/licenses`.
+
+The software is released under the GNU General Public License (GPL) which can be found in the file [`COPYING`](/COPYING) in the same directory as this file. Information about the licenses for the artwork is contained in `data/licenses`.
 
 ## 3D coordinates
-A reminder for those looking at the code and 3d models:
 
-STK    : X right, Y up,       Z forwards
+A reminder for those who are looking at the code & 3D models:
+
+SuperTuxKart: X right, Y up, Z forwards
 
 Blender: X right, Y forwards, Z up
 
-The exporters perform the needed transform, so in Blender you just work with XY plane as ground, and things will appear fine in STK (using XZ as ground in the code, obviously).
+The export utilities perform the needed transformation, so in Blender you just work with the XY plane as ground, and things will appear fine in STK (using XZ as ground in the code, obviously).
 
 ## Building from source
 
-First, you need both the code and the assets (See <https://supertuxkart.net/Source_control> for more information):
+In order to build SuperTuxKart from source, you'll need both the code and the assets (See <https://supertuxkart.net/Source_control> for more information):
 
-```
-git clone https://github.com/supertuxkart/stk-code
-svn checkout https://svn.code.sf.net/p/supertuxkart/code/stk-assets stk-assets
+```bash
+git clone https://github.com/supertuxkart/stk-code stk-code
```

Comment:
Redundant, git already creates a directory for the project based on its name, just saying.
Also, about svn, why using short command ?